charts/dapr/charts/dapr_placement/templates/dapr_placement_service.yaml

```diff
@@ -9,9 +9,14 @@ metadata:
     {{- range $key, $value := .Values.global.k8sLabels }}
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
-  {{- if .Values.service.annotations }}
+  {{- if or ( .Values.service.annotations ) ( eq .Values.global.prometheus.enabled true ) }}
   annotations:
   {{ toYaml .Values.service.annotations | indent 4}}
+  {{- if eq .Values.global.prometheus.enabled true }}
+    prometheus.io/scrape: "{{ .Values.global.prometheus.enabled }}"
+    prometheus.io/port: "{{ .Values.global.prometheus.port }}"
+    prometheus.io/path: "/"
```

Comment:
is this path correct for metrics?

Reply:
I copy pasted this from where it's being used on the deployments, so not sure

Reply:
the default is `/metrics` and why I asked. just double check or do not add it to use the default.

Reply:
will keep it as / for now for consistency

Reply:
My mantra is to avoid breaking changes and keeping consistency. Whatever is the setting that will give us that.